Steps4Impact/Journey/Cells/CurrentMilestoneCell.swift

```diff
@@ -0,0 +1,187 @@
+    }
+    containerRectangle.addSubview(milestoneCountLabel) {
+      $0.top.equalToSuperview().offset(Style.Padding.p4)
+      $0.leading.equalToSuperview().offset(Style.Padding.p12)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p12)
+      $0.height.equalTo(Style.Size.s24)
+    }
+    containerRectangle.addSubview(milestoneNameButton) {
+      $0.top.equalTo(milestoneCountLabel.snp.bottom).offset(Style.Padding.p2)
+      $0.leading.equalToSuperview().offset(Style.Padding.p12)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p12)
+      $0.height.equalTo(Style.Size.s24)
+    }
+    containerRectangle.addSubview(journeyMapImageView) {
+      $0.top.equalTo(milestoneNameButton.snp.bottom).offset(Style.Padding.p8)
+      $0.bottom.equalToSuperview().inset(Style.Padding.p16)
+      $0.leading.equalToSuperview().offset(Style.Padding.p12)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p12)
+      $0.height.width.equalTo(Style.Size.s128)
+    }
+    contentView.addSubview(containerRectangle) {
+      $0.top.equalToSuperview()
+      $0.leading.equalTo(circle.snp.trailing).offset(Style.Padding.p8)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p16)
+      $0.bottom.equalToSuperview().inset(Style.Padding.p16)
+    }
+    contentView.addSubview(verticalBar) {
+      $0.top.bottom.equalToSuperview()
+      $0.centerX.equalTo(circle.snp.centerX)
+      $0.width.equalTo(Style.Padding.p2)
+    }
+    contentView.addSubview(progressCircle) {
+      $0.top.equalToSuperview().offset(0)
+      $0.leading.equalToSuperview().offset(Style.Padding.p16)
+      $0.width.height.equalTo(Style.Size.s24)
+    }
+    contentView.addSubview(verticalProgressBar) {
+      $0.top.equalToSuperview()
+      $0.centerX.equalTo(circle.snp.centerX)
+      $0.width.equalTo(Style.Padding.p2)
+      $0.bottom.equalTo(progressCircle.snp.top)
+    }
+    contentView.bringSubviewToFront(circle)
+  }
+
+  func configure(context: CellContext) {
+
+    guard let currentMilestone = context as? CurrentMilestoneContext else { return }
+
+    let progressForCell = contentView.frame.height * (currentMilestone.progress/100)
+
+    progressCircle.alpha = 0
+
+    circle.backgroundColor = Style.Colors.FoundationGreen
+    verticalBar.backgroundColor = Style.Colors.Seperator
+
+    progressCircle.backgroundColor = Style.Colors.white
+    verticalProgressBar.backgroundColor = Style.Colors.FoundationGreen
+
+    if currentMilestone.sequence == 0 {
+      milestoneCountLabel.text = "Start"
+      milestoneNameButton.isHidden = true
+    } else {
+      milestoneNameButton.isHidden = false
+      milestoneCountLabel.text = "Milestone \(currentMilestone.sequence)/8"
+    }
+
+    if currentMilestone.sequence == 8 {
+      verticalBar.isHidden = true
+    } else {
+      verticalBar.isHidden = false
+    }
+    milestoneNameButton.setTitle("\(currentMilestone.name)", for: .normal)
+    
+    progressCircle.alpha = 1
+    UIView.animate(withDuration: 1) {
```

Comment:
I wouldn't call the animate in the configure method since it will be called for all cells when the tableview loads. If you want it to happen when the cell appears use UITableView's willDisplayCell method

Reply:
It'll just be called once. This animation is just within the CurrentMilestoneCell. Secondly, you can see the animation of the progress slider if it's called from willDisplayCell method